RealmSwift-swift2.0/LinkingObjects.swift

```diff
@@ -22,12 +22,33 @@ import Realm
 /// :nodoc:
 /// Internal class. Do not use directly. Used for reflection and initialization
 public class LinkingObjectsBase: NSObject, NSFastEnumeration {
-    internal var rlmResults: RLMResults
     internal let objectClassName: String
     internal let propertyName: String
 
-    init(results: RLMResults, fromClassName objectClassName: String, property propertyName: String) {
-        self.rlmResults = results
+    private var cachedRLMResults: RLMResults?
+    private var object: RLMWeakObjectHandle?
+    private var property: RLMProperty?
+
+    internal func attachTo(object object: RLMObjectBase, property: RLMProperty) {
+        self.object = RLMWeakObjectHandle(object: object)
+        self.property = property
+        self.cachedRLMResults = nil
+    }
+
+    internal var rlmResults: RLMResults {
+        if cachedRLMResults == nil {
+            if let object = self.object, let property = self.property {
```

Comment:
SwiftLint complains about this construct:
> warning: Conditional Binding Cascade Violation: Repeated `let` statements in conditional binding cascade should be avoided.

Any suggestions as to how cleanly avoid multiple `let` statements here?